abandoned_battlefield/map.conf

```
@@ -0,0 +1,13 @@
+initial_stuff = default:cobble 40, shooter:pistol, default:pick_steel, default:torch 5
```

```suggestion
initial_stuff = default:cobble 40,shooter:pistol,default:pick_steel,default:torch 5
```

Small fix, there shouldn't be spaces after comma otherwise initial stuff might not work correctly.